lib/cli/placeorder.ts

```diff
@@ -1,20 +1,21 @@
 import { Arguments, Argv } from 'yargs';
 import { Order, OrderSide, PlaceOrderEvent, PlaceOrderRequest, PlaceOrderResponse, SwapFailure, SwapSuccess } from '../proto/xudrpc_pb';
 import { callback, loadXudClient } from './command';
-import { coinsToSats, satsToCoinsStr } from './utils';
+import { checkPrecision, coinsToSats, satsToCoinsStr } from './utils';
 
 export const placeOrderBuilder = (argv: Argv, side: OrderSide) => {
   const command = side === OrderSide.BUY ? 'buy' : 'sell';
   argv.option('quantity', {
-    type: 'number',
+    type: 'string',
+    describe: 'the quantity for limit/market orders, max 12 digits on the right after a decimal point',
   })
   .option('pair_id', {
     type: 'string',
     describe: 'the trading pair ticker id for the order',
   })
   .option('price', {
     type: 'string',
-    describe: 'the price for limit orders, or "mkt"/"market" for market orders',
+    describe: 'the price for limit orders, or "mkt"/"market" for market orders, max 12 digits on the right after a decimal point',
```

Comment:
"max 12 decimal places" ? just a bit shorter that way and there's not much room in terminals